src/PoolManager.sol

```diff
@@ -124,9 +124,13 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
         returns (int24 tick)
     {
         // see TickBitmap.sol for overflow conditions that can arise from tick spacing being too large
-        if (key.tickSpacing > MAX_TICK_SPACING) TickSpacingTooLarge.selector.revertWith();
-        if (key.tickSpacing < MIN_TICK_SPACING) TickSpacingTooSmall.selector.revertWith();
-        if (key.currency0 >= key.currency1) CurrenciesOutOfOrderOrEqual.selector.revertWith();
+        if (key.tickSpacing > MAX_TICK_SPACING) TickSpacingTooLarge.selector.revertWith(key.tickSpacing);
+        if (key.tickSpacing < MIN_TICK_SPACING) TickSpacingTooSmall.selector.revertWith(key.tickSpacing);
+        if (key.currency0 >= key.currency1) {
+            CurrenciesOutOfOrderOrEqual.selector.revertWith(
+                Currency.unwrap(key.currency0), Currency.unwrap(key.currency1)
```

Comment:
How is this passing? The `CurrenciesOutOfOrderOrEqual` selector defined in IPoolManager seems unchanged

Reply:
good catch - its passing because we do low level reverts to save gas. fixing it now :)

Reply:
oh i see, its the same selector. still i guess good to update in the interface